TileFormats/FeatureTable/schema/featureTable.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "featureTable.schema.json",
+    "title" : "Feature Table",
+    "type" : "object",
+    "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
+    "patternProperties" : {
```

Comment:
I read that in the spec, but this schema doesn't, for example, explicitly disallow strings.

I would make the schema for the generic Feature Table as generic as possible.  Basically all we can say is that if a key is an object, and that object has a `byteOffset` property, then `byteOffset` should be:
```
"type" : "integer",
"minimum" : 0
```
Then let each tile format specifically scope what is valid for its Feature Table.  This is no more work for an implementation with our current tile formats, and allows future tile formats maximum flexibility to define their data structures.